app/components/admin/function_dropdown_component.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Admin
```

Comment:
I'm going to be extracting this for re-use in other admin function contexts in follow-up PR.